api/server.go

```diff
@@ -24,21 +24,18 @@ import (
 	"fmt"
 	"net/http"
 
-	"github.com/GeertJohan/go.rice"
```

Comment:
If this is removed, then it needs to be dropped from the `rice` command in `build-dist.sh` too.

Reply:
@liclac ready for liftoff!

![cat button press](http://iruntheinternet.com/lulzdump/images/cat-wearing-helmet-pressing-red-button-science-13977774880.gif)